services/121-service/src/notifications/sms/sms.service.ts

```diff
@@ -69,7 +69,13 @@ export class SmsService {
         messageContentType,
         messageProcessType,
       });
-      throw error;
+      if (error.code !== 21211) {
```

Comment:
```suggestion
      if (error.code !== TwilioErrorCodes.toNumberDoesNotExist) {
```

Maybe this one slipped through the cracks